### PR TITLE
`copyToClipboard`

### PR DESCRIPTION
**Pull request by hemanth**
Reduce the copy step wherever possible. (Haven't checked it cross browsers)

### CODE REVIEW

**Comment by hemanth** [comment]
Ping @jsoverson

File: repl.js [hunk 1]
```diff
@@ -426,6 +426,16 @@ window.onload = function () {
     })
   }
 
+  function copyToClipboard(value) {
+    var tempElem = document.createElement('input')
+    document.body.appendChild(tempElem);
+    tempElem.value = value;
+    tempElem.focus();
+    tempElem.setSelectionRange(0, tbx.value.length);
```
Comment (jsoverson) [inline repl.js:434]: I'm assuming tbx.value is supposed to be tempElem.value (or just value)?

**Comment by jsoverson** [comment]
This code doesn't work as is, and did you mean to add this somewhere else? This is added to the 'clear' button and also copies the state after the actual clear, neither of which seem to be very useful.

**Comment by hemanth** [comment]
Intention is to copy `.link`'s output to clip board.

**Comment by jsoverson** [comment]
Closing, feel free to resubmit. The code doesn't work as is.

**Pull request closed without merging**